tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "sourceRoot": "lib",
     "sourceMap": true,
     "baseUrl": "lib",
-    "allowJs": true
+    "allowJs": true,
+    // "checkJs": true
```

Comment:
lets make a decision on this

Reply:
Can't leave it in right now.  There still occurs one class of errors that I couldn't figure out how to resolve and didn't want to spend more time on.

Reply:
Regarding the remaining errors: I have confirmed they will go away when we move to TS.